pyproject.toml

```diff
@@ -3,7 +3,7 @@
 requires = [
     "setuptools",
     "wheel",
-    "Cython>=0.29.14",  # Note: keep in sync with tools/cythonize.py
+    "Cython>=0.29.19",  # Note: keep in sync with tools/cythonize.py
```

Comment:
There are also tests that use cython: `numpy/core/tests/test_cython.py` and `numpy/random/tests/test_extending.py`. I guess it would be prudent to refactor the three uses into a helper function...